### PR TITLE
fixed the bugs 

### PR DESCRIPTION
**Pull request by HelloMinh2122005**

### CODE REVIEW

**Comment by LyyHua** [comment]
Approved

File: SandTetris/Data/CheckInRepository.cs [hunk 1]
```diff
@@ -143,4 +144,58 @@ public async Task<IEnumerable<CheckIn>> GetCheckInsForEmployeeAsync(string emplo
...
+                checkInSummary.TotalOnLeave++;
+            }
+            else if (newStatus == CheckInStatus.Absent)
+            {
+                checkInSummary.TotalAbsent++;
+            }
+
+            await databaseService.DataContext.SaveChangesAsync();
+        }
+    }
```
Comment (honguyenminh) [inline SandTetris/Data/CheckInRepository.cs:194]: This method does literally nothing. Why is it here?

**Review by honguyenminh — COMMENTED** [review]
Fix it. Fast.

File: SandTetris/Data/CheckInRepository.cs (hunk 1 above)
Comment (HelloMinh2122005) [reply SandTetris/Data/CheckInRepository.cs:194]: It seems like my bad for forgetting to delete it. Sorry bro, i'll fix it

Comment (LyyHua) [reply SandTetris/Data/CheckInRepository.cs:194]: Go kill yourself

**Pull request merged**